src/ui/components/App.tsx

```diff
@@ -107,8 +107,17 @@ function App({ children, modal, theme }: AppProps) {
     // Stop space bar from being used as a universal "scroll down" operator
     // We have a big play/pause interface, so space makes way more sense for that.
     window.addEventListener("keydown", function (e) {
-      if (e.code === "Space" && e.target === document.body) {
-        e.preventDefault();
+      if (e.code === "Space") {
+        const codeMirrorScroll = document.querySelectorAll(".CodeMirror-scroll");
+        let isCodeMirror = false;
+        for (let el of codeMirrorScroll) {
+          if (el === e.target) {
+            isCodeMirror = true;
+          }
+        }
```

Comment:
could this be `const isCodeMirror = e.target.classList.has(.CodeMirror-scroll)`

Reply:
Yep, tsc was yelling at me so I was taking evasive maneuvers but got there eventually 😆